tests/Kernel/fixtures/rendering.yml

```diff
@@ -775,28 +798,41 @@
       title: "Item title"
       url: "http://example.com"
       detail: "Item text"
+      lenght: 10
       image:
         src: "http://via.placeholder.com/120x80"
         alt: "Alternate text for secondary image"
       meta:
         - "Meta 1"
         - "Meta 2"
+      field_list:
+        - label: "Countries"
+          body: "Spain, France, Italy"
+        - label: "Deadline dates"
+          body: "1 April 2017 / 1 July 2017"
   assertions:
     count:
       'article.ecl-content-item div.ecl-content-item__title a.ecl-link[href="http://example.com"]': 1
       'article.ecl-content-item div[aria-label="Alternate text for secondary image"].ecl-u-d-lg-block': 1
       'article.ecl-content-item div.ecl-content-item__image__before': 0
       'article.ecl-content-item div.ecl-content-item__image__after': 1
+      'article>div .ecl-description-list .ecl-description-list__term': 2
+      'article>div .ecl-description-list .ecl-description-list__definition': 2
     equals:
       'article.ecl-content-item div.ecl-content-item__meta': "Meta 1 | Meta 2"
       'article.ecl-content-item div.ecl-content-item__title .ecl-link': "Item title"
       'article.ecl-content-item div.ecl-content-item__description': "Item text"
+      'article>div .ecl-description-list .ecl-description-list__term:nth-child(1)': "Countries"
+      'article>div .ecl-description-list .ecl-description-list__definition:nth-child(2)': "Spain, France, Italy"
+      'article>div .ecl-description-list .ecl-description-list__term:nth-child(3)': "Deadline dates"
+      'article>div .ecl-description-list .ecl-description-list__definition:nth-child(4)': "1 April 2017 / 1 July 2017"
```

Comment:
Let's put spaces around `>` so it's easier to read, like this:
```
       'article > div .ecl-description-list .ecl-description-list__definition:nth-child(4)': "1 April 2017 / 1 July 2017"
```

Reply:
Updated